src/Exception/LogicException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * This file is part of the Global Trading Technologies Ltd doctrine-auditable-bundle package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace Gtt\Bundle\DoctrineAuditableBundle\Exception;
+
+/**
+ * {@inheritdoc}
+ */
+class LogicException extends \LogicException implements DoctrineAuditableExceptionInterface
```

Comment:
You **MUST NOT** use the same exception class name as default ones, because it's quite unpleasant to trigger autocompletion and check every possible namespace for choosing a correct one `LogicException`. Give it better name.